ecl/hqlcpp/hqlsource.cpp

```diff
@@ -7275,6 +7276,8 @@ ABoundActivity * HqlCppTranslator::doBuildActivityFetch(BuildCtx & ctx, IHqlExpr
         return info.buildActivity(ctx, expr, TAKcsvfetch, "CsvFetch", childActivity);
     case no_xml:
         return info.buildActivity(ctx, expr, TAKxmlfetch, "XmlFetch", childActivity);
+    case no_json:
+        return info.buildActivity(ctx, expr, TAKjsonfetch, "XmlFetch", childActivity);
```

Comment:
I'll add a comment, because I still want to have the generated code take advantage of the existing xmlFetch classes,